packages/inline-banner/tests/integration/components/nucleus-inline-banner-test.js

```diff
@@ -69,4 +70,12 @@ module('Integration | Component | nucleus-inline-banner', function(hooks) {
       assert.ok(true, 'no a11y errors found!');
     });
   });
+
+  test('it passes visual regression tests', async function(assert){
+    await render(hbs`{{nucleus-inline-banner
```

Comment:
Can we add a couple of variants addressing other props to this component?

Reply:
Added as part of the latest commit.